pom.xml

```diff
@@ -154,6 +154,21 @@
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>bigquery-connector</artifactId>
       <version>${bigquery.connector.hadoop2.version}</version>
+      <exclusions>
+        <!-- Dependencies are excluded because they are missing in central Maven repository -->
```

Comment:
weird, so it doesn't compile unless you exclude these?

Reply:
Correct.